app/shared/ophanTypes.ts

```diff
@@ -0,0 +1,73 @@
+export type OphanProduct =
+  | "CONTRIBUTION"
+  | "RECURRING_CONTRIBUTION"
+  | "MEMBERSHIP_SUPPORTER"
+  | "MEMBERSHIP_PATRON"
+  | "MEMBERSHIP_PARTNER"
+  | "DIGITAL_SUBSCRIPTION"
+  | "PAPER_SUBSCRIPTION_EVERYDAY"
+  | "PAPER_SUBSCRIPTION_SIXDAY"
+  | "PAPER_SUBSCRIPTION_WEEKEND"
+  | "PAPER_SUBSCRIPTION_SUNDAY"
+  | "PRINT_SUBSCRIPTION"
+  | "APP_PREMIUM_TIER";
+
+export type OphanAction =
+  | "INSERT"
+  | "VIEW"
+  | "EXPAND"
+  | "LIKE"
+  | "DISLIKE"
+  | "SUBSCRIBE"
+  | "ANSWER"
+  | "VOTE"
+  | "CLICK";
+
+export type OphanComponentType =
+  | "READERS_QUESTIONS_ATOM"
```

Comment:
Do we need all of these component types in the repo? They don't all seem that relevant to manage-frontend.